lib/index.ts

```diff
@@ -32,10 +33,10 @@ type ContextFn = (...args: any[]) => ElementHandle
 
 async function createElementHandleArray(handle: JSHandle): Promise<ElementHandle[]> {
   const lengthHandle = await handle.getProperty('length')
-  const length = await lengthHandle.jsonValue()
+  const length = await lengthHandle.jsonValue() as number
 
   const elements: ElementHandle[] = []
-  for (let i = 0; i < length; i++) {
+  for (let i = 0; i < length; i += 1) {
```

Comment:
is there seriously a rule for this in tslint? 🤦‍♂

Reply:
There is. I was trying to avoid adding exceptions to rules but it might be better to add the exceptions and make plans to migrate to eslint later.

Reply:
SGTM!